### PR TITLE
Adding trips#show

### PR DESCRIPTION
**Pull request by adam-conway**
Changes proposed in this pull request:  
- This adds trip#show functionality for tests, controllers, and views
- Added in validation to the trip model which I forgot to do when creating trip#index

@Vadlusk @agpiermarini @anubiskhan - Ready to be reviewed!

connect to #7

### CODE REVIEW

File: app/controllers/trips_controller.rb [hunk 1]
```diff
@@ -2,4 +2,8 @@ class TripsController < ApplicationController
   def index
     @trips = Trip.all.paginate(page: params[:page], per_page: 30)
```
Comment (Vadlusk) [inline app/controllers/trips_controller.rb]: Is that really all there is to pagination? It has to be more complex than this...

File: app/views/trips/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<%= @trip.duration %>
```
Comment (Vadlusk) [inline app/views/trips/show.html.erb]: I think this is a good place for a table, and that all three of the indexes we're working on tonight will have similar tables.

File: app/controllers/trips_controller.rb (hunk 1 above)
Comment (adam-conway) [reply app/controllers/trips_controller.rb]: That's really all there is! You also have to added <%= will_paginate @trips %> to the index page so you can scroll between pages.

File: app/views/trips/show.html.erb (hunk 2 above)
Comment (adam-conway) [reply app/views/trips/show.html.erb]: Yeah I agree. I figured we'd wait on creating a default table structure for all the indexes until we had some sort of wireframe or consensus on how we want it to look. This at least is the data and is pretty easy to add a table to it later.

Comment (Vadlusk) [reply app/views/trips/show.html.erb]: That's a good call. After I finish the second user story, I'll draw up some ideas for wireframes. If we all do the same, we can have a good discussion.

**Comment by Vadlusk** [comment]
This all looks good and straight-forward.  I'm not sure all the validations are necessary, but if they are, I need to add them to the condition model as well.

**Pull request merged**